Sources/BrowserServicesKit/ContentBlocking/UserScripts/surrogates.js

```diff
@@ -21,7 +21,7 @@
     const duckduckgoDebugMessaging = (function () {
         let log = () => {}
         let signpostEvent = () => {}
-
+        
```

Comment:
Can you run `git checkout origin/main -- Sources/BrowserServicesKit/ContentBlocking/UserScripts/surrogates.js` or similar?